pset-functions.js

```diff
@@ -136,6 +185,52 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const scoreTable = {
```

Comment:
@jacobkelman good job using an object to map the letters to their scores

be careful with your indentation here